Makefile

```diff
@@ -236,10 +229,12 @@ $(LIBC_SO): $(call timestamp_of,c-cpp-linking) \
 				-shared \
 				-o $(OUTPUT_DIR)/profiles/$(shell basename $(d))/lib/libc.so \
 				*.c $(KCCFLAGS) -I .)
-	$(info $(PROFILE): Done translating the C standard library.)
+	@$(LOGGER) "$(PROFILE): Done translating the C standard library."
 
 
 .PHONY: $(PROFILE)-native
+$(PROFILE)-native: CC := $(PROFILE_DIR)/cc
+$(PROFILE)-native: CXX := $(PROFILE_DIR)/cxx
```

Comment:
This is the added code relevant to https://github.com/kframework/c-semantics/pull/500/files#r297394004